src/main/java/core/basesyntax/strategy/impl/PurchaseOperationHandlerImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class PurchaseOperationHandlerImpl implements OperationHandler {
+    @Override
+    public int count(FruitTransaction fruitTransaction) {
```

Comment:
I think your handler should make rurchase operation and here you should place a check if fruit balance is positive or zero.